src/together/types/finetune.py

```diff
@@ -320,3 +323,27 @@ class FinetuneLRScheduler(BaseModel):
 
 class FinetuneLinearLRSchedulerArgs(BaseModel):
     min_lr_ratio: float | None = 0.0
+
+
+class FinetuneCheckpoint(BaseModel):
+    """
+    Fine-tune checkpoint information
+    """
+
+    # checkpoint type (e.g. "Intermediate", "Final", "Final Merged", "Final Adapter")
+    type: str
+    # timestamp when the checkpoint was created
+    timestamp: str
+    # checkpoint name/identifier
+    name: str
+
+
+class FinetuneCheckpointList(BaseModel):
+    """
+    List of fine-tune checkpoints
+    """
+
+    # object type
+    object: Literal["list"] | None = None
```

Comment:
It doesn't look like you need this field, can you remove it? In fact, do you really need to define this class, or just passing around `List[FinetuneCheckpoint]` will do?

Reply:
Removed the object